scene/gui/tab_container.cpp

```diff
@@ -584,8 +584,8 @@ void TabContainer::remove_child_notify(Node *p_child) {
 		return;
 	}
 
-	Control *c = Object::cast_to<Control>(p_child);
-	if (!c || c->is_set_as_top_level()) {
+	Control *c = as_sortable_control(p_child, SortableVisbilityMode::VISIBLE);
```

Comment:
```suggestion
	Control *c = as_sortable_control(p_child, SortableVisbilityMode::IGNORE);
```

This should be ignore to match previous behavior and match `add_child_notify`.